runtime/balrt.h

```diff
@@ -305,6 +305,24 @@ static READONLY inline int64_t taggedFloatCompare(TaggedPtr lhs, TaggedPtr rhs)
     return taggedPrimitiveCompare(lhs, rhs, &taggedFloatComparator);
 }
 
+static READONLY inline int64_t taggedBooleanComparator(TaggedPtr lhs, TaggedPtr rhs) {
+    int lhsVal = taggedToBoolean(lhs);
+    int rhsVal = taggedToBoolean(rhs);
+    if (lhsVal == rhsVal) {
+        return COMPARE_EQ;
+    }
+    if (lhsVal < rhsVal) {
+        return COMPARE_LT;
+    }
+    if (lhsVal > rhsVal) {
+        return COMPARE_GT;
+    }
+    return COMPARE_UN;
```

Comment:
Boolean can't be `COMPARE_UN`, so above `if` should be an `else`.

Reply:
Fixed